Src/Base/AMReX_ParallelDescriptor.H

```diff
@@ -107,7 +107,7 @@ while ( false )
     void Initialize ();
     void Finalize ();
 
-    extern AMREX_EXPORT int use_gpu_aware_mpi;
+    extern AMREX_EXPORT bool use_gpu_aware_mpi;
```

Comment:
Casting from int to bool is well defined. So I don't think we need to make any changes here. Is your compiler complaining about it?

Reply:
Actually, there is no problem compiling the code, but if I use "amrex.use_gpu_aware_mpi=False" according to the docs, there will be a parmparse error.

Reply:
I've recently tested the GPU-aware MPI feature on a platform that uses Hygon DCU (AMD arch gfx906), and the result is not very satisfying.

Reply:
The documentation is incorrect. It should be 0 or 1.

Reply:
Even if the type is bool, the flag needs to be `false`, not `False`. So I will fix the documentation. If we change the type here to bool, we need to do that to its definition in AMReX_ParallelDescriptor.cpp too.

Reply:
#3562

Reply:
By the way, using `1` or `0` is always right for ParmParse boolean type parameters whether the C++ type is `int` or `bool`.

Reply:
Yeah, ok. So we just need to correct the documentation.